Makefile

```diff
@@ -107,6 +110,9 @@ vet: ## Run go vet against code.
 #	test -f ${ENVTEST_ASSETS_DIR}/setup-envtest.sh || curl -sSLo ${ENVTEST_ASSETS_DIR}/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/v0.8.3/hack/setup-envtest.sh
 #	source ${ENVTEST_ASSETS_DIR}/setup-envtest.sh; fetch_envtest_tools $(ENVTEST_ASSETS_DIR); setup_envtest_env $(ENVTEST_ASSETS_DIR); go test ./... -coverprofile cover.out
 
+.PHONY: ci-job
+ci-job: common-deps-update generate fmt fmt-code vet bundle-check 	# todo: golangci-lint unittest shellcheck bashate bundle-check
```

Comment:
add `manifests` after `generate`

Reply:
we already have ci/prow/gofmt and ci/prow/govet jobs. We might want to run them as separate jobs so that one failure does not stop others to run.

Reply:
👍🏻

Reply:
maybe move the whole thing under `bundle-check` and call the job bundle-check? The ci-job is too general, we want to have different jobs run in parallel.